app/components/DocContainer.tsx

```diff
@@ -9,7 +9,7 @@ export function DocContainer({
     <div
       {...props}
       className={twMerge(
-        'w-full max-w-full space-y-2 md:space-y-6 lg:space-y-8 p-2 md:p-6 lg:p-8',
```

Comment:
This spacing was only being used to add margin to the "Back" button on the blog page. So it was moved over to the button itself as the `space-` utils would otherwise have been applied to the`<TocMobile>` component even if it was visually hidden on the screen.

I opted not to use a React (`useMediaQuery`) hook in-favor of relying on CSS.